CODEOWNERS

```diff
@@ -80,10 +80,12 @@ go.mod @fleetdm/go
 #
 # (see website/config/custom.js for DRIs of other paths not listed here)
 ##############################################################################################
-/handbook/company/product-groups.md                   @mikermcneil
-/handbook/company/why-this-way.md                     @mikermcneil
-/handbook/company/README.md                           @mikermcneil
-/handbook/README.md                                   @mikermcneil  # « This is the "Table of contents" landing page
+/handbook                   @mikermcneil  # Everyone can contribute to the handbook, and page DRIs are expected to keep their pages up to date.    
```

Comment:
"See https://github.com/fleetdm/fleet/pull/13195/files"
Instead of dup comment.